src/codeFly/frontEnd/about.html

```diff
@@ -11,8 +11,8 @@
     <title>About Us</title>
     <!-- Bootstrap -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-    <link href="/css/mystyles.css" rel="stylesheet">
-    <link href="/css/bootstrap-social.css" rel="stylesheet">
+    <link href="css/mystyles.css" rel="stylesheet">
+    <link href="css/bootstrap-social.css" rel="stylesheet">
```

Comment:
Keep the absolute paths as they are. Otherwise our server can't send correct files to the browser.